### DISCUSSION TITLE
Missing Instructions to Add 'use client'

### DISCUSSION THREAD

**Discussion by ItsANoBrainer** [vercel/community#5653] [Bug Report]
### Page to Investigate

https://nextjs.org/learn/dashboard-app/mutating-data#2-create-a-server-action

### Steps to Reproduce

In the section "Create a Server Action", the create-form.tsx will error/crash and not start unless you add the `'use client';`. 

The tutorial should either instruct the user to add it and explain why, or it should be added to the [starter-example](https://github.com/vercel/next-learn/blob/main/dashboard/starter-example/app/ui/invoices/create-form.tsx).

The [final-example](https://github.com/vercel/next-learn/blob/main/dashboard/final-example/app/ui/invoices/create-form.tsx#L1) does have this in it.

**Comment by amyegan** [comment]
Thanks for reporting this! I'll share this info with the Next.js team so we can get it fixed

**Comment by delbaoliveira** [comment]
Hey @ItsANoBrainer, could you share the error and your code? 

You shouldn't have to add `"use client"` until [Chapter 14 when adding `useFormState`](https://nextjs.org/learn/dashboard-app/improving-accessibility#:~:text=In%20your%20create%2Dform.tsx%20component%2C%20import%20the%20useFormState%20hook%20from%20react%2Ddom.%20Since%20useFormState%20is%20a%20hook%2C%20you%20will%20need%20to%20turn%20your%20form%20into%20a%20Client%20Component%20using%20%22use%20client%22%20directive%3A). This is to show forms can be Server Components. But there might be something else causing the error.
